src/main/kotlin/com/jobtracker/api/repository/entities/ContactEntity.kt

```diff
@@ -0,0 +1,29 @@
+package com.jobtracker.api.repository.entities
+
+import java.util.UUID
+import javax.persistence.*
+
+@Entity
+@Table(name = "contacts")
+data class ContactEntity(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    val id: UUID,
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    // Not sure how to set get this as the company's name rather than ID.
+    // Citation: https://www.baeldung.com/jpa-one-to-one section 3.2
+    // Ommitted from Company because assuming this is unidrectional relationship
+    @JoinColumn(name = "company_name", referencedColumnName = "id") 
```

Comment:
I don't think we want to store the company name here. I think we want to store the company_id. Later when we need the company name, we can use a `join` to get the company name from the companies table.